<launch>
    <param name="prefix" value="R_" />
    <node pkg="seed_robotics" name="seed_hand" type="hand_handle_node.py" output="screen">
      <rosparam file="$(find seed_robotics)/config/RH4D.yaml" />
    </node>

<!-- Uncomment this if you want to use right hand sensors. WARN : Remember to edit ../config/sensors_right.yaml file
    <node pkg="sensor_pkg" name="seed_fts3_right" type="read_publish_sensor_node.py" output="screen">
    	<rosparam file="$(find sensor_pkg)/config/sensors_right.yaml" />
    </node>
-->
</launch>
